mise.toml

```diff
@@ -1,3 +1,47 @@
 [tools]
 node = "22.11.0"
 pnpm = "9.15.3"
+
+[tasks."postgres:eql:download"]
+alias = 'e'
+description = "Download latest EQL release"
+outputs = [
+  "{{config_root}}/sql/cipherstash-encrypt.sql",
+  "{{config_root}}/sql/cipherstash-encrypt-uninstall.sql",
+]
+run = """
+mkdir sql
+
+# install script
+if [ -z "$CS_EQL_PATH" ]; then
+  curl -sLo sql/cipherstash-encrypt.sql https://github.com/cipherstash/encrypt-query-language/releases/latest/download/cipherstash-encrypt.sql
+else
+  echo "Using EQL: ${CS_EQL_PATH}"
+  cp "$CS_EQL_PATH" sql/cipherstash-encrypt.sql
+fi
+
+# uninstall script
+if [ -z "$CS_EQL_UNINSTALL_PATH" ]; then
+  curl -sLo sql/cipherstash-encrypt-uninstall.sql https://github.com/cipherstash/encrypt-query-language/releases/latest/download/cipherstash-encrypt-uninstall.sql
+else
+  echo "Using EQL: ${CS_EQL_PATH}"
+  cp "$CS_EQL_UNINSTALL_PATH" sql/cipherstash-encrypt-uninstall.sql
+fi
+"""
+
+[tasks."postgres:setup"]
```

Comment:
I figured you might rework this to be more idiomatic for a JS project @calvinbrewer , but here's how to install EQL with the mise task:
```
docker compose up -d
mise run postgres:setup
```

Reply:
This was lifted from Proxy.